cblite/EnrichCommand.cc

```diff
@@ -88,17 +87,36 @@ void EnrichCommand::enrichDocs(const string& srcProp, const string& dstProp, uni
     if (_offset > 0)
         cout << "(Skipping first " << _offset << " docs)\n";
     
-    // Start transaction
-    C4Error error;
-    c4::Transaction t(_db);
-    if (!t.begin(&error))
-        fail("Couldn't open database transaction");
+    map<string, MutableDict> docDict;
+    vector<Value> props;
+    int64_t nDocs = 0;
     
-    // Loop through and enrich docs
+    tie(docDict, props, nDocs) = readData(srcProp, options);
```

Comment:
This made me realize that if nDocs is zero here, there is no point in going further.

Reply:
This is not resolved.  If there are zero docs here, why bother trying to proceed past this line?

Reply:
readData updates the number of docs

Reply:
Inside readData there is a fail if nDocs is 0. I can move it into enrichDocs if needed.

Reply:
Oh, if there is a fail inside there that's ok then.  In that case you won't need this at the bottom that appears to be there now:

```
    if (nDocs == 0) {
        cout << "(No documents with property matching \"" << srcProp << "\"" << ")";
        fail("No documents to enrich");
    }
```